packages/mdc-menu/simple/foundation.js

```diff
@@ -83,6 +83,7 @@ export default class MDCSimpleMenuFoundation extends MDCFoundation {
     this.scaleY_ = 0;
     this.running_ = false;
     this.selectedTriggerTimerId_ = 0;
+    this.animationRequestId_ = null;
```

Comment:
nit: since this will always be a non-zero, non-negative integer, it's probably best just to set this to `0`. This also means you can get rid of the `if` statement within destroy and just call `cancelAnimationFrame`. It also keeps it consistent with `selectedTriggerTimerId_`.